rapidast.py

```diff
@@ -444,5 +472,49 @@ def run():
         sys.exit(0)
 
 
+def merge_sarif_files(directory: str, properties: dict, output_filename: str):
+    """
+    Searches for .sarif and .sarif.json files in a directory, merges them,
+    and adds "properties" as properties to the merged SARIF log
+
+    Args:
+        directory: The directory to search for SARIF files
+        properties: A dictionary containing arbitrary properties for each scanner
+        output_filename: The name of the output merged SARIF file
+    """
+    merged_runs = []
+    for filename in os.listdir(directory):
+        if filename.endswith(".sarif") or filename.endswith(".sarif.json"):
+            filepath = os.path.join(directory, filename)
+            logging.info(f"Found SARIF file: {filepath}")
+            try:
+                with open(filepath, "r", encoding="utf8") as f:
+                    data = json.load(f)
+                    if "runs" in data and isinstance(data["runs"], list):
+                        merged_runs.extend(data["runs"])
+                    else:
+                        logging.warning(f"SARIF file '{filepath}' does not appear to have a top-level 'runs' array.")
+
+            except Exception as e:  # pylint: disable=W0718
+                logging.error(f"Error reading SARIF file '{filepath}': {e}")
+
+    if not merged_runs:
+        logging.warning(f"No SARIF files found in directory: {directory}")
+
+    merged_sarif = {
```

Comment:
I believe we want to add the Rapidast version as well

Reply:
Config version is already added  is this line:
```
sarif_properties = {"config_version": config.get("config.configVersion"), "scanner_results": scanner_results}
```

Reply:
We discussed this internally and decided that we want the RapiDAST version in addition to the config one. That said, it won’t be easy to get since it’s only available in Git at the moment. So, we’ll include it as part of a separate feature and PR